lib/sync-settings.coffee

```diff
@@ -53,7 +53,9 @@ module.exports =
       cb?(err, res)
 
   getPackages: ->
-    for name,info of atom.packages.getLoadedPackages()
+    packages = atom.packages.getLoadedPackages()
+    for name,info of packages
+      continue unless packages.hasOwnProperty(name)
```

Comment:
Nope. It generates exactly what we need:
```javascript
var info, name, packages;
packages = atom.packages.getLoadedPackages();
for (name in packages) {
  info = packages[name];
  if (!packages.hasOwnProperty(name)) {
    continue;
  }
}
```
Actually, right now I found in Coffeescript documentation more clean way (it makes same check):
```coffee
for own name, info of packages
  ...
```